web/controllers/task_controller.ex

```diff
@@ -40,12 +41,22 @@ defmodule CodeCorps.TaskController do
   def record(_conn, id), do: Task |> Repo.get(id)
 
   @spec handle_create(Plug.Conn.t, map) :: Ecto.Changeset.t
-  def handle_create(_conn, attributes) do
+  def handle_create(conn, attributes) do
+    project = Project |> Repo.get(attributes["project_id"])
+    if project.github_id do
+      current_user = Guardian.Plug.current_resource(conn)
+      github_id = github_module().create_issue(attributes, project, current_user)
+      attributes = Map.merge(attributes, %{"github_id" => github_id})
+    end
```

Comment:
Hey @begedin, I initially did that and was wondering the reasoning for pulling this out into a service? There really isnt a lot of complicated logic (most of that happens in the Github service) and wondering why we would create a task service just to copy the lines we have here? Are we thinking of using it beyond the CRUD actions? thanks!